README.md

```diff
@@ -76,8 +76,8 @@ nano ~/.zshrc
 
   ```bash
 
-  # use lower case GitHub User Name as branch name
-  export MY_BRANCH=$(echo $GITHUB_USER | tr '[:upper:]' '[:lower:]')
+  # by default MY_BRANCH is set to your lower case GitHub User Name
+  # the value can be overwritten if needed
   echo $MY_BRANCH
```

Comment:
These are repeated on `outerloop-multicluster` and `outerloop-aks` labs, should we update those as well?